config/config.hpp.in

```diff
@@ -112,6 +112,19 @@
 // Enable MFEM functionality based on the PUMI library
 // #define MFEM_USE_PUMI
 
+// Build the GPU/CUDA-enabled version of the MFEM library.
+// Requires a CUDA compiler (nvcc).
+// #define MFEM_USE_CUDA
```

Comment:
Good to see this option was added! I was worried that non-experts wouldn't be able to build with GPU support.

Reply:
`make cuda`